packages/components/src/Airswap/AirswapBuyCVL.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from "react";
+import makeAsyncScriptLoader from "react-async-script";
+import { Button } from "../Button";
+
+const AIRSWAP_URL = "https://cdn.airswap.io/gallery/airswap-trader.js";
```

Comment:
Hmm, we should maybe put these configs into `@joincivil/utils` along with other config / network derived values.

Reply:
I will add this adding a TODO for now